scripts/street_sign_recognizer.py

```diff
@@ -17,10 +19,29 @@ def __init__(self):
         """ Initialize the street sign reocgnizer """
         rospy.init_node('street_sign_recognizer')
         self.cv_image = None                        # the latest image from the camera
+        self.cv_image_res = None
+        self.grayscale_sign = None
         self.bridge = CvBridge()                    # used to convert ROS messages to OpenCV
         cv2.namedWindow('video_window')
+        cv2.namedWindow('grayscale_sign_window')
         rospy.Subscriber("/camera/image_raw", Image, self.process_image)
 
+        cv2.namedWindow('threshold_image')
+        self.hsv_lb = np.array([30, 75, 75]) # hsv lower bound
+        cv2.createTrackbar('H lb', 'threshold_image', 0, 360, self.set_h_lb)
```

Comment:
Trackbars are nice, and I like that you put in your default values so you didn't have to set them every time you ran the code.  Dynamic reconfigure is also a nice way to do this, and there are some tools for saving values between runs